docs/site/plugins/mdx-link-append-loader.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (source) {
+  const filePath = this.resourcePath;
```

Comment:
This needs to use `path.relative()` with the root directory of the project to account for CI build paths